### ISSUE TITLE
Card embed block names

### ISSUE THREAD

**Issue by cavasinf** [kevinpapst/TablerBundle#120]
Big BC BREAK but we should rename blocks inside card embed.

https://github.com/kevinpapst/TablerBundle/blob/a2d91a62db826e770c645515286615668673cf1f/templates/embeds/card.html.twig#L24

They are called `box_xxxxx` as a remnant of AdminLTE. And classes `card-xxxx`
We should stick to Bootstrap and/or Tabler name convention and use `card_xxxx` instead.

WDYT ?

**Comment by kevinpapst** [comment]
Yeah, I think that is true: but I hate the idea, considering in how many files I use these block names ^^
But we should write a wrapper that is able to handle that name change for existing installations...

**Comment by cavasinf** [comment]
A "replace in files" `CTRL + SHIFT + R` from " box_" to " card_" (with a space at the beginning) should be enough? 😄 

102+ ref on my side 😭 
![image](https://user-images.githubusercontent.com/25293190/193806614-e49c9e9d-354f-46ca-856e-808aba5298b7.png)


> But we should write a wrapper that is able to handle that name change for existing installations...

And AdminLTEBundle Migration..

**Comment by kevinpapst** [comment]
Likely yes, but: my software supports plugins and there are many, which are not under my control... but they will have to adapt to the new theme anyway 😁 so good to go from my side.

**Comment by kevinpapst** [comment]
Changed the milestone, because this has to be done BEFORE the first "real" release.

**Comment by wucherpfennig** [comment]
Please just make sure to notify me before I update ;-) well... I will find out anyway ^^
